docs/browser.md

```diff
@@ -78,16 +84,16 @@ the following changes:
  // to use when deploying to other networks.
 -const HARDHAT_NETWORK_ID = '1337';
 +const HARDHAT_NETWORK_ID = '23295'; // Sapphire Testnet
```

Comment:
`SAPPHIRE_TESTNET_NETWORK_ID`?

Reply:
This relates to a specific tutorial which modifies the hardhat boilerplate project to use Sapphire. Modifying that is out of the scope of this PR.